### DISCUSSION TITLE
AI降临 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#127] [Announcements]
# AI降临 · 构建我的被动收入

当AI降临了，我们该怎么办？

https://www.bmpi.dev/self/ai-arrival/

**Comment by xifengxx** [comment]
这几天也尝试了使用ChatGPT，真的感觉是智能AI，一旦功能完善并商用起来，真的有概率改变现有的互联网产业版图。

**Comment by madawei2699** [comment]
ChatGPT 正在杀死编程里的乐趣：https://www.zlovezl.cn/articles/chatgpt-and-how-we-programming/

**Reply by madawei2699** [reply]
不止会杀死编程的乐趣，可能也会杀死很多专业程序员的工作，人类通过程序语言与计算机对话变成了与AI用自然语言对话，编程的门槛可能也变的荡然无存。

**Comment by madawei2699** [comment]
一篇分析 ChatGPT 背后技术的深度好文：[通向AGI之路：大型语言模型（LLM）技术精要](https://zhuanlan.zhihu.com/p/597586623)。

**Comment by madawei2699** [comment]
AI 替代绝大部分人工活动后会怎么样？罗兰·T. 拉斯特给的一个答案是“情感”。他在《情感经济：人工智能、颠覆性变革与人类未来》一书里，把人类的经济活动分成三类：

1.体力经济时代——靠劳动力来赚钱的方式，体力至上的男性，是这个时期的主导者，逐渐被机械替代；

2.思维经济时代——信息技术革命，人类进入以服务业为主导地位的脑力时代，“脑子好的人”成为这个时代的宠儿。

3.情感经济时代——人工智能逐渐开始比人类更聪明之后，人类能发挥作用的地方，会逐渐转移到情感领域。
